sycl/test-e2e/Regression/host_image_linear.cpp

```diff
@@ -0,0 +1,216 @@
+// RUN: %clangxx -fsycl -fsycl-targets=%sycl_triple %s -o %t.out
+// RUN: %BE_RUN_PLACEHOLDER %t.out
+
+// Tests that host-side SYCL 1.2.1 image accessors correctly read the right
+// value when accessing using linear sampling mode.
```

Comment:
Should we compare the host accessor against a device one?

Reply:
We could, but not only would it make a heavier test, we would also create an aspect requirement for this test which we do not have currently.